src/tests/test_utils/common_test_utils/include/common_test_utils/postgres_helpers.hpp

```diff
@@ -3,6 +3,7 @@
 //
 
 #pragma once
+
```

Comment:
Please remove extra line to avoid recompilation the file

Reply:
done